app/assets/stylesheets/profile-page.scss

```diff
@@ -0,0 +1,10 @@
+@import 'colors';
+
+.image-card {
+
+    img {
+        max-width: 56%;
+        max-height: 18rem;
+        border-radius: 25rem;
+      }
+}
```

Comment:
you'll want to check your tab size settings - looks like this file is a tab size of 4 and we want a consistent tab size of 2 for all team members. this can be edited in your settings within vscode